### PR TITLE
Components newslleter, footer and bloglist

### PR DESCRIPTION
**Pull request by EliveltonSouzaDev**
feat: components

### CODE REVIEW

**Review by n3ndor — APPROVED** [review]

File: src/app/components/blogList.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// src/components/ListItem.tsx
```
Comment (paulpessoa) [inline src/app/components/blogList.tsx]: remove coment

File: src/app/components/blogList.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+'use client'
+
+import React from 'react';
+import { FaRegPlayCircle } from 'react-icons/fa';
```
Comment (paulpessoa) [inline src/app/components/blogList.tsx:4]: replace for hero icons

File: src/app/components/blogList.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        subtitle: string;
+        icon: string;
+        minutes: number;
+    }[];
+}
+
+const ListItem: React.FC<ListItemProps> = ({ items }) => {
+    return (
+        <div className="container bg-white py-2 px-4 border rounded-lg w-80">
+            <h1 className="text-xl font-semibold mb-4 text-gray-800">Articles</h1>
```
Comment (paulpessoa) [inline src/app/components/blogList.tsx:18]: padding top can add more space

File: src/app/components/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// src/components/Footer.tsx
```
Comment (paulpessoa) [inline src/app/components/footer.tsx]: remove coment

File: src/app/components/footer.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import React from 'react';
+import { FaGithub, FaFacebook, FaTwitter, FaYoutube } from 'react-icons/fa';
```
Comment (paulpessoa) [inline src/app/components/footer.tsx:2]: replace to hero icons , you are using "fa- font awesome"

File: src/app/components/footer.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+const Footer: React.FC = () => {
+    return (
+        <footer className="bg-white text-black py-4 absolute bottom-0 w-full">
+            <div className="container mx-auto flex items-center justify-between">
+                <div>
+                    EcoBagain GmbH © 2023 All rights reserved.
```
Comment (paulpessoa) [inline src/app/components/footer.tsx]: we can add a js function to keep updated the "year"

File: src/app/components/modalTerms.tsx
Comment (paulpessoa) [inline src/app/components/modalTerms.tsx:1]: good idea to create this terms

File: src/app/components/modalTerms.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+                <br />  Only administrators have access to the email address on Pelando, and these will never be shared with third parties unless required by law.
+              </p>
+            </div>
+            <div className="flex items-center justify-end p-6 border-t border-solid border-slate-200 rounded-b">
+              <button
+                className="text-red-500 background-transparent font-bold uppercase px-6 py-2 text-sm outline-none focus:outline-none mr-1 mb-1 ease-linear transition-all duration-150"
+                type="button"
+                onClick={closeModal}
+              >
+                Close
```
Comment (paulpessoa) [inline src/app/components/modalTerms.tsx:51]: the buttons could follow the tailwind UI, and the prototype colors.

for example the color teal instead green, and Capitalize instead UPERCASE

File: src/app/components/newsletter.tsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+              <form className='flex flex-col items-center' onSubmit={handleSubmit}>
+                <input
+                  type="email"
+                  placeholder="Enter your email address here"
+                  value={email}
+                  onChange={handleInput}
+                  className="p-2 text-base text-gray-700 border-2 border-gray-300 rounded-md"
+                />
+
+                <button
```
Comment (paulpessoa) [inline src/app/components/newsletter.tsx:56]: adjust color button

File: src/app/components/newsletter.tsx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+                  className="m-2 p-2 w-40 bg-green-500 border-none text-white text-base rounded-md cursor-pointer"
+                >
+                  Subscribe
+                </button>
+              </form>
+
+            </div>
+            <div className="my-4 lg:mb-0">
+              <p className="text-md text-blue-500 hover:text-blue-700 cursor-pointer">
+                <span onClick={() => setShowModal(true)}> Terms of Service and Privacy Policy</span>
```
Comment (paulpessoa) [inline src/app/components/newsletter.tsx:67]: adjust color terms

File: src/app/page.tsx [hunk 10]
```diff
@@ -1,9 +1,32 @@
-import ItemsList from './components/ItemsList'
+import Newsletter from 'components/newsletter'
+import ListItem from 'components/blogList'
```
Comment (paulpessoa) [inline src/app/page.tsx]: adjust this as it is confusing to import a component called "blogList" and use it as "ListItem"
It's very generic

**Review by paulpessoa — CHANGES_REQUESTED** [review]

File: src/app/components/blogList.tsx (hunk 2 above)
Comment (EliveltonSouzaDev) [reply src/app/components/blogList.tsx:4]: I couldn't find this social media icons on hero Icons, that's why i got from font awesome

**Review by paulpessoa — APPROVED** [review]

**Pull request merged**